packages/connect/src/SmoldotProvider/SmoldotProvider.test.ts

```diff
@@ -85,7 +112,8 @@ test('emits events when it connects then disconnects', async () => {
   });
 });
 
-test('emits events when it connects / disconnects / reconnects', async () => {
+/** Skip as a potential issue tests (see: https://github.com/paritytech/substrate-connect/issues/325#issuecomment-862585814) */
+test.skip('emits events when it connects / disconnects / reconnects', async () => {
```

Comment:
You skipped the wrong test.  'emits events when it connects / disconnects / reconnects' is definitely a valid useful test. We want to skip 'emits connect and never emits disconnect for development chain'  - although I actually think better to just remove it entirely since its not valuable. WE can get it back from git history if we think we want it later.  This

Reply:
Yes my bad - the other test is fixed now. Will try to fix this and have no deleted or skipped tests